src/workspaces/dashboard/BucketLocation.ts

```diff
@@ -60,6 +58,9 @@ export const BucketLocation = requesterPaysWrapper({ onDismiss: _.noop })((props
         // while storageDetails.googleBucketLocation will contain the default value.
         // In the case of requester pays workspaces, we wish to show the user more information in this case and allow them to link a workspace.
         loadGoogleBucketLocation();
+      } else if (storageDetails.fetchedGoogleBucketLocation === 'RPERROR') {
+        setNeedsRequesterPaysProject(true);
+        setLoading(false);
```

Comment:
added a case here for `RPERROR` which sets the proper state. the `ERROR` case continues to try to reload; this might happen as permissions are being synced.